test/core/address_utils/parse_address_with_named_scope_id_test.cc

```diff
@@ -43,21 +45,22 @@ static void test_grpc_parse_ipv6_parity_with_getaddrinfo(
   absl::StatusOr<grpc_core::URI> uri = grpc_core::URI::Parse(target);
   if (!uri.ok()) {
     gpr_log(GPR_ERROR, "%s", uri.status().ToString().c_str());
-    GPR_ASSERT(uri.ok());
+    ASSERT_TRUE(uri.ok());
   }
   grpc_resolved_address addr;
-  GPR_ASSERT(1 == grpc_parse_ipv6(*uri, &addr));
+  ASSERT_EQ(1, grpc_parse_ipv6(*uri, &addr));
```

Comment:
Should be `  ASSERT_TRUE(grpc_parse_ipv6(*uri, &addr));`

Reply:
Done.